Functions/xASL_adm_Load4DMemMapping.m

```diff
@@ -19,10 +19,10 @@
 
 
 %% Admin, specify for each modality how to treat it
-ModalitiesList  = {'ASL' 'ASL_notScaled' 'ASL_untreated' 'ASL_PVEc'  'ASL_HctCohort'    'ASL_HctCorrInd' 'PWI' 'M0' 'mean_control' 'SD' 'SNR' 'TT' 'PV_pGM' 'PV_pWM'  'FLAIR'   'T1'  'WMH_SEGM'  'c1T1' 'R1'};

```

Comment:
This wrapper/function can be phased out, we don't use it anymore

Reply:
This function:
Development/StatsAnalysis/SPM_VBA_Stats/xASL_wrp_spm_GLM.m
was still using it. So I have deleted the ASL_adm_Load4DMemMapping.m file and moved the function to that wrapper. Can be then deleted with it. But can't delete function that is referenced from, even an obscure one, function.

Reply:
Yes nice. This spm wrapper is quite useful for doing voxel-wise GLM analyses